Gemfile

```diff
@@ -6,12 +6,13 @@ gemspec
 gem 'engine_cart'
 
 gem 'rubocop', require: false
-
+gem 'yajl-ruby', require: 'yajl'
```

Comment:
@anarchivist I know this is already merged, but can you say something about this?  Will the correct thing get required when this is installed by a host app?